README.md

```diff
@@ -4,53 +4,65 @@
 
 ---
 
-# Getting started (For people who aren't used to Discord.js)
+# Getting started
 
+## Requirements
 
-### Requirements
-* Node.js
-* A code editor (visual studio code, atom, notepad++ etc)
-* Git
+- [Node.js](https://nodejs.org)
+- Any text editor (Ex. [Visual Studio Code](https://code.visualstudio.com/), [Atom](https://atom.io/), [WebStorm](https://www.jetbrains.com/webstorm/))
+- [Git](https://git-scm.com/downloads)
```

Comment:
This link is only for windows. Linux have to install it by shell, so only `Git` is needed.